pkg/controllers/nodeclaim/garbagecollection/nodeclaim_test.go

```diff
@@ -356,4 +358,17 @@ var _ = Describe("NodeClaim/GarbageCollection", func() {
 		}
 		wg.Wait()
 	})
+	It("should not delete an instance if EnableNodePools/EnableNodeClaims isn't enabled", func() {
+		nodepoolutil.EnableNodePools = false
+		nodeclaimutil.EnableNodeClaims = false
+
+		// Launch time was 10m ago
+		instance.LaunchTime = aws.Time(time.Now().Add(-time.Minute))
```

Comment:
Does it need to be this? Or comment changed?
```suggestion
		instance.LaunchTime = aws.Time(time.Now().Add(-10 * time.Minute))
```

Reply:
Comment was wrong. Just updated